pipelinedp4j/tests/com/google/privacy/differentialprivacy/pipelinedp4j/spark/SparkTests.kt

```diff
@@ -16,10 +16,25 @@
 
 package com.google.privacy.differentialprivacy.pipelinedp4j.spark
 
+import org.apache.spark.sql.SparkSession
 import org.junit.runner.RunWith
 import org.junit.runners.Suite
 
 /** Provides a list of JUnit test classes to Bazel. When creating a new test class, add it here. */
 @RunWith(Suite::class)
 @Suite.SuiteClasses(SparkCollectionTest::class, SparkEncodersTest::class, SparkTableTest::class)
 class SparkTests {}
+
+// Helper method to create SparkSession
+fun createSparkSession(): SparkSession {
+    return SparkSession.builder()
+        .appName("Kotlin Spark Example")
+        .master("local[*]")
+        .config("spark.driver.bindAddress", "127.0.0.1")
+        .getOrCreate()
+}
+
+// Helper method to stop SparkSession
+fun stopSparkSession(spark: SparkSession) {
```

Comment:
but we don't call `stopSparkSession`.

actually we can create a JUnit Rule that will call it after every test: https://chatgpt.com/share/67424bad-578c-8003-8ef9-de498f534b62.

Reply:
thanks for suggesting. Created JUnit Rule.